### PR TITLE
transport-crash-report-creation

### PR DESCRIPTION
**Pull request by 03fernanda-palacios**
Transport implementation of ACRA

### CODE REVIEW

File: BundleTransport/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
 
 
-    <application
+    <application android:name=".utils.acraUtils"
```
Comment (breed) [inline BundleTransport/app/src/main/AndroidManifest.xml:13]: we should probably put this on it's own line like the rest of the attributes

File: BundleTransport/app/src/main/java/net/discdd/bundletransport/utils/LocalReportSender.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    @Override
+    public void send(Context context, CrashReportData errorContent) throws ReportSenderException {
+        // Iterate over the CrashReportData instance and do whatever
+        // you need with each pair of ReportField key / String value
+
+        //declare root dir and append target dir (toServer)
+        Path rootDir = context.getApplicationContext().getExternalFilesDir(null).toPath();
+        Path destDir = rootDir.resolve("BundleTransmission/server");
+        if (destDir.toFile().exists()) {
+            File logFile = new File(String.valueOf(destDir), "crash_report.txt");
```
Comment (breed) [inline BundleTransport/app/src/main/java/net/discdd/bundletransport/utils/LocalReportSender.java:33]: this line is repeated in the else, you should pull it out and put it below the if

Comment (breed) [reply BundleTransport/app/src/main/java/net/discdd/bundletransport/utils/LocalReportSender.java:33]: oh actually it is already there. you can just remove this line and the other in the else clause

File: BundleTransport/app/src/main/java/net/discdd/bundletransport/utils/LocalReportSender.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        }
+        File logFile = new File(String.valueOf(destDir), "crash_report.txt");
+        try {
+            // Use the core ReportFormat configuration
+            String reportText = config.getReportFormat()
+                    .toFormattedString(errorContent,
+                                       config.getReportContent(), "\n", "\n\t", false);
+
+            // Overwrite last report
+            FileWriter writer = new FileWriter(logFile, false);
```
Comment (breed) [inline BundleTransport/app/src/main/java/net/discdd/bundletransport/utils/LocalReportSender.java:46]: this is fine for this PR, but we should probably change it to keep the last 5 or something.

File: BundleTransport/app/src/main/java/net/discdd/bundletransport/utils/acraUtils.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
+package net.discdd.bundletransport.utils;
```
Comment (breed) [inline BundleTransport/app/src/main/java/net/discdd/bundletransport/utils/acraUtils.java:1]: when you do BundleClient, you probably want to move this class to bundle-core

**Review by breed — APPROVED** [review]
just a couple of small changes.

File: BundleTransport/app/src/main/java/net/discdd/bundletransport/utils/LocalReportSender.java (hunk 3 above)
Comment (03fernanda-palacios) [reply BundleTransport/app/src/main/java/net/discdd/bundletransport/utils/LocalReportSender.java:46]: I forgot to ask you about that- Good to know thank you!

**Pull request merged**